### DISCUSSION TITLE
Response Headers

### DISCUSSION THREAD

**Discussion by Syscomp-Software** [OpenSourceCommunityBrasil/REST-DataWare#238] [General]
Gostaria de saber a maneira correta de adicionar headers de resposta, utilizei desta maneira 
` Params.RequestHeaders.Output.Add('teste=1');`, 
mas não obtive sucesso.

**Comment by mobius1qwe** [comment]
tanto no recebimento quanto na resposta você vai usar Params, você precisa criar o param, atribuir valor pra ele que vai ser devolvido na resposta
